AutoCollection/Statsbeat.ts

```diff
@@ -174,36 +179,31 @@ class Statsbeat {
                 "language": this._language,
                 "version": this._sdkVersion,
                 "attach": this._attach,
-                "instrumentation": this._instrumentations,
             }
             this._trackRequestDuration(networkProperties);
             this._trackRequestsCount(networkProperties);
-            let attachProperties = {
-                "os": this._os,
-                "rp": this._resourceProvider,
-                "rpid": this._resourceIdentifier,
-                "cikey": this._cikey,
-                "runtimeVersion": this._runtimeVersion,
-                "language": this._language,
-                "version": this._sdkVersion
-            }
-            this._statbeatMetrics.push({ name: Constants.StatsbeatCounter.ATTACH, value: 1, properties: attachProperties });
             await this._sendStatsbeats();
         });
     }
 
     public async trackLongIntervalStatsbeats() {
         this._getResourceProvider(async () => {
-            let featureProperties = {
+            let commonProperties = {
                 "os": this._os,
                 "rp": this._resourceProvider,
                 "cikey": this._cikey,
                 "runtimeVersion": this._runtimeVersion,
                 "language": this._language,
                 "version": this._sdkVersion,
                 "attach": this._attach,
-                "feature": this._features,
-            }
+            };
+            let featureProperties = Object.assign({ "feature": this._features }, commonProperties);
+            let instrumentationProperties = Object.assign({ "instrumentation": this._instrumentations }, commonProperties);
+            let attachProperties = Object.assign({
+                "rpid": this._resourceIdentifier,
+            }, commonProperties);
+            this._statbeatMetrics.push({ name: Constants.StatsbeatCounter.ATTACH, value: 1, properties: attachProperties });
+            this._statbeatMetrics.push({ name: Constants.StatsbeatCounter.INSTRUMENTATION, value: 1, properties: instrumentationProperties });
```

Comment:
I don't think it's confirmed yet to create a new instrumentation type.. we need to reconsider customDimentions for this.  can you provide the payload for it? same as feature?

Reply:
yep same as feature but with instrumentation property instead, let me know if we need to update the properties here

Reply:
think about it more, we might not need to create a new metric for instrumentation.. we can reuse the feature metric.. they're identical. maybe just rename the "feature" to 'encodedLong' to support both feature and instrumentation and then add a flag to indicate type of encodedLong.

Reply:
we need to talk though.